src/media.css

```diff
@@ -39,13 +39,61 @@
     justify-content: center;
   }
 
-  .p-reset {
+  .pad-reset {
     padding: 0 !important;
   }
 
-  .m-reset {
+  .mar-reset {
     margin: 0 !important;
   }
+
+  .mar-sm {
+    --media: var(--media-sm);
+  }
+
+  .mar-md {
+    --media: var(--media-md);
+  }
+
+  .mar-lg {
+    --media: var(--media-lg);
+  }
+
+  .mar-xl {
+    --media: var(--media-xl);
+  }
+
+  .pad-sm {
+    --media: var(--media-sm);
+  }
+
+  .pad-md {
+    --media: var(--media-md);
+  }
+
+  .pad-lg {
+    --media: var(--media-lg);
+  }
+
+  .pad-xl {
+    --media: var(--media-xl);
+  }
+
+  .h-sm {
```

Comment:
should this be `height-sm` or something similar?

Reply:
`h` is confusing, have changed to `hdg` since it's a heading multiplier